Polyhedron/demo/Polyhedron/Plugins/PCA/Basic_generator_plugin.cpp

```diff
@@ -350,8 +350,11 @@ void Basic_generator_plugin::generateCube()
 
     for(int i=0; i<8; ++i)
     {
-
+#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
       QStringList list = point_texts[i].split(QRegExp("\\s+"), QString::SkipEmptyParts);
+#else
+      QStringList list = point_texts[i].split(QRegExp("\\s+"), Qt::SkipEmptyParts);
+#endif
```

Comment:
I suggested you define a macro, somewhere, that evaluates to `QString::SkipEmptyParts` or `Qt::SkipEmptyParts`. That would factorize all those occurrences of `#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)` to one place: the definition of the macro.